Tests/RZHostAppTests/RZAutoLayoutHelpersTest.m

```diff
@@ -0,0 +1,227 @@
+    XCTAssertEqualWithAccuracy(CGRectGetMidY(frameTwo), 500.0f, RZAutoLayoutHelpersTestAllowableDelta);
+}
+
+- (void)testCenteringWithOffsetViews
+{
+    [self.testViewOne rz_centerHorizontallyInContainerWithOffset:RZAutoLayoutHelpersTestCenterOffset];
+    [self.testViewOne rz_centerVerticallyInContainerWithOffset:RZAutoLayoutHelpersTestCenterOffset];
+    [self.testViewTwo rz_centerHorizontallyInContainerWithOffset:RZAutoLayoutHelpersTestCenterOffset];
+    [self.testViewTwo rz_centerVerticallyInContainerWithOffset:RZAutoLayoutHelpersTestCenterOffset];
+    
+    [self forceConstraintsEvaluation];
+    
+    XCTAssert([self.testViewOne rz_pinnedCenterXConstraint] != nil);
+    XCTAssert([self.testViewOne rz_pinnedCenterYConstraint] != nil);
+    XCTAssert([self.testViewTwo rz_pinnedCenterXConstraint] != nil);
+    XCTAssert([self.testViewTwo rz_pinnedCenterYConstraint] != nil);
+    
+    CGRect frameOne = self.testViewOne.frame;
+    
+    XCTAssertEqualWithAccuracy(frameOne.origin.x, 397.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameOne.origin.y, 397.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameOne.size.width, 250.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameOne.size.height, 250.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    
+    XCTAssertEqualWithAccuracy(CGRectGetMidX(frameOne), 522.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(CGRectGetMidY(frameOne), 522.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    
+    CGRect frameTwo = self.testViewTwo.frame;
+    
+    XCTAssertEqualWithAccuracy(frameTwo.origin.x, 434.5f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameTwo.origin.y, 357.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameTwo.size.width, 175.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(frameTwo.size.height, 330.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    
+    XCTAssertEqualWithAccuracy(CGRectGetMidX(frameTwo), 522.0f, RZAutoLayoutHelpersTestAllowableDelta);
+    XCTAssertEqualWithAccuracy(CGRectGetMidY(frameTwo), 522.0f, RZAutoLayoutHelpersTestAllowableDelta);
+}
+
+- (void)testPinningViews {
+    
+    [self.testViewOne rz_pinTopSpaceToSuperviewWithPadding:0.0f];
+    [self.testViewOne rz_pinRightSpaceToSuperviewWithPadding:0.0f];
+    [self.testViewOne rz_pinBottomSpaceToSuperviewWithPadding:0.0f];
+    [self.testViewOne rz_pinLeftSpaceToSuperviewWithPadding:0.0f];
+    
+    [self.testViewTwo rz_pinTopSpaceToSuperviewWithPadding:21.0f];
+    [self.testViewTwo rz_pinRightSpaceToSuperviewWithPadding:21.0f];
+    [self.testViewTwo rz_pinBottomSpaceToSuperviewWithPadding:17.0f];
+    [self.testViewTwo rz_pinLeftSpaceToSuperviewWithPadding:17.0f];
+    
+    [self forceConstraintsEvaluation];
+    
+    XCTAssert([self.testViewOne rz_pinnedTopConstraint] != nil);
```

Comment:
I recommend using `XCTAssertNotNil` here, since it gives better logs when it fails.